src/modules/dashboard/components/InputStorageWidget/InputStorageWidget.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+import * as yup from 'yup';
+
+import { ActionTypes } from '~redux/index';
+import { ActionForm } from '~core/Fields';
+
+import InputStorageWidgetForm from './InputStorageWidgetForm';
+
+export interface FormValues {
+  inputStorageLot?: string;
+  inputNewValue?: string;
+}
+
+const InputStorageWidget = () => {
+  const validationSchema = yup.object().shape({
+    inputStorageLot: yup.string().required(),
+    inputNewValue: yup.string().required(),
+  });
+
+  return (
+    <ActionForm
+      initialValues={{
+        inputStorageLot: undefined,
+        inputNewValue: undefined,
```

Comment:
```suggestion
        inputStorageLot: '',
        inputNewValue: '',
```

Reply:
When you change this, you won't get that: "input field uncontrolled error anymore"

![Screenshot from 2020-12-08 15-58-02](https://user-images.githubusercontent.com/1193222/101493031-2d0dd880-396e-11eb-823c-002d16d3205f.png)